Moggles/ClientApp/src/featureToggle/EditFeatureToggle.vue

```diff
@@ -136,6 +136,18 @@
                 this.getEnvironments();
             });
         },
+        computed: {
+            confirm() {
+                let environmentsEditedNumber = 0;
```

Comment:
There is no need to explicitly say it's a number, or maybe _numberOfEditedEnvironments_ ? The word order changes everything 😄